src/strings/en-US/PowerFxResources.resw

```diff
@@ -1,45 +1,104 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
What happened here?

Reply:
weird, not sure how this got changed, will revert this change.

Reply:
still an issue?

Reply:
yeah, it's changing that on every commit.